src/api/ThreadedComments.js

```diff
@@ -18,12 +18,24 @@ import {
     ERROR_CODE_CREATE_REPLY,
 } from '../constants';
 
-import type { ThreadedCommentPermission } from '../common/types/threadedComments';
 import type { ElementsXhrError, ElementsErrorCallback } from '../common/types/api';
 import type { BoxItem, BoxItemPermission } from '../common/types/core';
-import type { FeedItemStatus } from '../common/types/feed';
+import type { BoxCommentPermission, Comment, FeedItemStatus } from '../common/types/feed';
 
 class ThreadedComments extends MarkerBasedApi {
+    /**
+     * Formats comment data for use in components.
+     *
+     * @param {Comment} comment - An individual comment entry from the API
+     * @return {Comment} Updated comment
+     */
+    format(comment: Comment): Comment {
+        return {
```

Comment:
Should we also format replies?

Reply:
Good point, I will actually add that in a future PR that will cover logic changes related to replies.